vms/platformvm/block/proposal_block_test.go

```diff
@@ -46,6 +46,7 @@ func TestNewBanffProposalBlock(t *testing.T) {
 		parentID,
 		height,
 		tx,
+		[]*txs.Tx{},
```

Comment:
We should test this field being set like the others below

Reply:
Added a test